### DISCUSSION TITLE
create PacketSender multiple instances.

### DISCUSSION THREAD

**Discussion by freewong** [dannagle/PacketSender#333] [Ideas]
can I start multiple PacketSender instances? I need to listen multiple different ports.

**Comment by dannagle** [comment]
Yes, you can launch multiple instances provided you don't create a port conflict. You can achieve this by having the listening port be 0.

Alternatively, you can use one Packet Sender and listen to multiple ports with it. Just separate them by commas. 


Example listening to 3 ports on UDP and TCP:

![image](https://github.com/dannagle/PacketSender/assets/1140573/c5f39866-573e-4c84-8f30-245282d0cd15)


You can see it bound on the bottom right:
![image](https://github.com/dannagle/PacketSender/assets/1140573/e57fc8af-2877-408a-a695-e55df9b4a267)


Hope that helps,
Dan
